test/integration/goldens/redis/BUILD.bazel

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "redis_goldens_files",
```

Comment:
Just `goldens_files` should be sufficient, since this is in the `redis` directory.

Reply:
SG, renamed